src/js/big-play-button.js

```diff
@@ -36,6 +36,11 @@ class BigPlayButton extends Button {
   handleClick(event) {
     const playPromise = this.player_.play();
 
+    // exit early if clicked via the mouse
+    if (this._mouseused && event.clientX && event.clientY) {
```

Comment:
Could we just use `clientX` or `clientY` without `mouseused` here? could be a terrible idea but I figured I would throw it out there.